proposals/1.2-proposal.md

```diff
@@ -0,0 +1,113 @@
+<div align="center">
+    <h1>Log ASCII Proposal v1.2</h1>
+</div>
+
+<h2> Overview of various fields</h2>
+
+~V (Version Information)
+
+- This section is **mandatory** and usually appears at the very beginning of the file.
+- It identifies which version of the LAS format is being used and whether
+wrap mode is used.
+
+    - If wrap mode is off - Max length on line 256 chars
+    - If wrap mode in on - Max length on line 80 chars
+
+```
+-----------------------------------------------------------------------------
+~Version Information Section
+VERS.             1.2:   CWLS log ASCII Standard -VERSION 1.2
+WRAP.               NO:   One line per depth step
+-----------------------------------------------------------------------------
+```
+**Proposed Data struct - Dictionary**
+
+~W (Well Information)
+
+- This section is **mandatory**.
+- It identifies the well, its location and indicates the start and stop depths of the file.Basically every information that is there for the description of well
+- This segment contains a lot of **mnemonics**
+```
+-----------------------------------------------------------------------------
+~Well Information Section
+#MNEM.UNIT	Data Type	Information  
+#---------	-------------	------------------------------
+STRT.M           635.0000:
+STOP.M           400.0000:
+STEP.M            -0.1250:
+NULL.             -999.25:
+COMP.             COMPANY:      ANY OIL COMPANY INC.
+WELL.                WELL:      ANY ET AL A9-16-49-20
+FLD .               FIELD:      EDAM
+LOC .            LOCATION:      A9-16-49-20W3M
+PROV.            PROVINCE:      SASKATCHEWAN
+SRVC.     SERVICE COMPANY:      ANY LOGGING COMPANY INC.
+DATE.            LOG DATE:      13-DEC-86
+UWI .      UNIQUE WELL ID:      100091604920W300
+-----------------------------------------------------------------------------
+```
+**Proposed Data Struct- Table with MNEM, unit data type and Info as columns**
+
+~C (Curve Information)
+- This basically contains all type of measurements of the particular well that were taken, ie, contains the list of all the possible values that are there in the data table.
+- By possible values, it is meant that type of measurements like - Depth, Bulk Density, etc.
+- The curves described in this section must be present in the data set.
+```
+-----------------------------------------------------------------------------
+~Curve Information Section
+#MNEM.UNIT	API CODE	Curve Description	
+#---------    -------------   -------------------------------
+DEPTH.M                      :    1       DEPTH
+RHOB .K/M3        7 350 02 00:    2       BULK DENSITY
```

Comment:
What are these values `7 350 02 00`? I am assuming that they are the `API CODE` according to the table header. Is this code documented properly?

Reply:
Yes. This is API Code. We may or may not keep it. I think it's just for cross reference

Reply:
How is this code used? Let's keep it because it may be useful for the person working with the well.